src/js/__tests__/__snapshots__/typecheck-test.ts.snap

```diff
@@ -3313,6 +3226,7 @@ plugins/services/src/js/reducers/serviceForm/JSONReducers/Volumes.ts: error TS26
 plugins/services/src/js/reducers/serviceForm/JSONReducers/Volumes.ts: error TS2683: 'this' implicitly has type 'any' because it does not have a type annotation.
 plugins/services/src/js/reducers/serviceForm/JSONReducers/Volumes.ts: error TS2683: 'this' implicitly has type 'any' because it does not have a type annotation.
 plugins/services/src/js/reducers/serviceForm/JSONReducers/Volumes.ts: error TS2683: 'this' implicitly has type 'any' because it does not have a type annotation.
+plugins/services/src/js/reducers/serviceForm/JSONReducers/Volumes.ts: error TS2683: 'this' implicitly has type 'any' because it does not have a type annotation.
```

Comment:
i've manually checked all the additions in here. some of them just moved within the file and the other ones only appear because the form now has some typing at all.